docker-compose.yml

```diff
@@ -26,13 +26,13 @@ services:
         - LOGGING_USE_JSON=0
         - MISSION_CONTROL_TABLE=error_aggregates
       command:
-        ./bin/run-dev.sh
+        "web-dev"
```

Comment:
It's configured like atmo to use run the ENTRYPOINT: https://github.com/mozilla/missioncontrol/pull/10/files#diff-3254677a7917c6c01f55212f86c57fbfR38